pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -453,9 +487,9 @@ func (m *ManagerImpl) readCheckpoint() error {
 	m.podDevices.fromCheckpointData(data.PodDeviceEntries)
 	m.allocatedDevices = m.podDevices.devices()
 	for resource, devices := range data.RegisteredDevices {
-		m.allDevices[resource] = sets.NewString()
+		m.healthyDevices[resource] = sets.NewString()
```

Comment:
Could you add a TODO comment on considering to also checkpoint unhealthy device information?